src/modules/45-dashboards/pages/home/HomePage.tsx

```diff
@@ -563,311 +574,270 @@ const HomePage: React.FC = () => {
+              items={filteredDashboardList}
+              renderItem={(dashboard: DashboardInterface) => (
+                <Card className={cx(css.dashboardCard)}>
+                  <Container>
+                    {(dashboard?.type === dashboardType.SHARED || dashboard?.type === dashboardType.ACCOUNT) && (
+                      <CardBody.Menu
+                        menuContent={
+                          <Menu>
+                            <MenuItem text="clone" onClick={() => clone(dashboard.id)} />
+                          </Menu>
+                        }
+                        menuPopoverProps={{
+                          className: Classes.DARK
+                        }}
+                      />
+                    )}
 
-        {!loading && (
-          <Layout.Vertical padding={{ left: 'medium', right: 'medium' }}>
-            <Pagination
-              itemCount={100}
-              pageSize={10}
-              pageCount={100}
-              pageIndex={page}
-              gotoPage={(pageNumber: number) => setPage(pageNumber)}
+                    <Layout.Vertical
+                      spacing="large"
+                      onClick={() => {
+                        history.push({
+                          pathname: routes.toViewCustomDashboard({
+                            viewId: dashboard.id,
+                            accountId: accountId,
+                            folderId: folderId === 'shared' ? 'shared' : dashboard?.resourceIdentifier
+                          })
+                        })
+                      }}
+                    >
+                      <Text color={Color.BLACK_100} font={{ size: 'normal', weight: 'semi-bold' }}>
+                        {dashboard?.title}
+                      </Text>
+                      {TagsRenderer(dashboard)}
+
+                      <Layout.Horizontal spacing="medium">
+                        {dashboard?.type !== dashboardType.SHARED && (
+                          <>
+                            <Layout.Horizontal style={{ marginRight: 'var(--spacing-8)' }}>
+                              <Icon name="eye-open" size={18} style={{ marginRight: 'var(--spacing-4)' }} />
+                              <Text color={Color.BLACK_100} font={{ size: 'normal', weight: 'semi-bold' }}>
+                                {dashboard?.view_count}
+                              </Text>
+                            </Layout.Horizontal>
+
+                            <Layout.Horizontal>
+                              <Icon name="star-empty" size={18} style={{ marginRight: 'var(--spacing-4)' }} />
+                              <Text color={Color.BLACK_100} font={{ size: 'normal', weight: 'semi-bold' }}>
+                                {dashboard?.favorite_count}
+                              </Text>
+                            </Layout.Horizontal>
+                          </>
+                        )}
+                      </Layout.Horizontal>
```

Comment:
Any reason not to move the outer layout inside the condition?
```suggestion
                      
                        {dashboard?.type !== dashboardType.SHARED && (
                          <Layout.Horizontal spacing="medium">
                            <Layout.Horizontal style={{ marginRight: 'var(--spacing-8)' }}>
                              <Icon name="eye-open" size={18} style={{ marginRight: 'var(--spacing-4)' }} />
                              <Text color={Color.BLACK_100} font={{ size: 'normal', weight: 'semi-bold' }}>
                                {dashboard?.view_count}
                              </Text>
                            </Layout.Horizontal>

                            <Layout.Horizontal>
                              <Icon name="star-empty" size={18} style={{ marginRight: 'var(--spacing-4)' }} />
                              <Text color={Color.BLACK_100} font={{ size: 'normal', weight: 'semi-bold' }}>
                                {dashboard?.favorite_count}
                              </Text>
                            </Layout.Horizontal>
                          </Layout.Horizontal>
                        )}
```